clients/src/main/java/org/apache/kafka/common/config/SaslConfigs.java

```diff
@@ -32,6 +32,11 @@
         + "Only GSSAPI is enabled by default.";
     public static final List<String> DEFAULT_SASL_ENABLED_MECHANISMS = Collections.singletonList(GSSAPI_MECHANISM);
 
+    /** Jaas configuration file format is described <a href="http://docs.oracle.com/javase/7/docs/technotes/guides/security/jgss/tutorials/LoginConfigFile.html">here</a> */
+    public static final String SASL_JAAS_CONFIG = "sasl.jaas.config";
```

Comment:
@rajinisivaram can we not provide keytab an principal name instead of passing the jaas config. Its still another config management todo for the users.